commit 2c18433a4cf22e73706866a341a536ef2d1f1e97
Author: bertin <macportsuser@macports.org>
Date:   Fri Oct 10 19:16:20 2025 +0200

    /opt/local/linux-ports/audio/audacity/files/patch-portaudio-cmake-alias.diff

diff --git a/cmake-proxies/cmake-modules/FindPortAudio.cmake b/cmake-proxies/cmake-modules/FindPortAudio.cmake
index fb03fadb3..85cfd95ba 100644
--- a/cmake-proxies/cmake-modules/FindPortAudio.cmake
+++ b/cmake-proxies/cmake-modules/FindPortAudio.cmake
@@ -15,13 +15,13 @@ if( NOT PortAudio_FOUND )
          message( STATUS "Found PortAudio: \n\tPortAudio_INCLUDE_DIR: ${PortAudio_INCLUDE_DIR}\n\tPortAudio_LIBRARIES: ${PortAudio_LIBRARIES}" )
       endif()
 
-      if( NOT TARGET portaudio::portaudio )
-         add_library( portaudio::portaudio INTERFACE IMPORTED GLOBAL)
+      if( NOT TARGET PortAudio::PortAudio )
+         add_library( PortAudio::PortAudio INTERFACE IMPORTED GLOBAL)
 
-         target_include_directories( portaudio::portaudio INTERFACE ${PortAudio_INCLUDE_DIR} )
-         target_link_libraries( portaudio::portaudio INTERFACE ${PortAudio_LIBRARIES} )
+         target_include_directories( PortAudio::PortAudio INTERFACE ${PortAudio_INCLUDE_DIR} )
+         target_link_libraries( PortAudio::PortAudio INTERFACE ${PortAudio_LIBRARIES} )
 
-         add_library(portaudio ALIAS portaudio::portaudio)
+         add_library(portaudio ALIAS PortAudio::PortAudio)
       endif()
    else()
       if( PortAudio_FIND_REQUIRED )
diff --git a/lib-src/libnyquist/CMakeLists.txt b/lib-src/libnyquist/CMakeLists.txt
index 59fa18e18..6ffe7ddbf 100644
--- a/lib-src/libnyquist/CMakeLists.txt
+++ b/lib-src/libnyquist/CMakeLists.txt
@@ -295,7 +295,7 @@ list( APPEND OPTIONS
 
 list( APPEND LIBRARIES
    PRIVATE
-      portaudio::portaudio
+      PortAudio::PortAudio
       SndFile::sndfile
 )
 
diff --git a/lib-src/portmixer/CMakeLists.txt b/lib-src/portmixer/CMakeLists.txt
index 589fa66d8..65a384fa0 100644
--- a/lib-src/portmixer/CMakeLists.txt
+++ b/lib-src/portmixer/CMakeLists.txt
@@ -6,7 +6,7 @@ def_vars()
 
 if (UNIX AND NOT APPLE)
    include(CheckIncludeFile)
-   set(CMAKE_REQUIRED_LIBRARIES portaudio::portaudio)
+   set(CMAKE_REQUIRED_LIBRARIES PortAudio::PortAudio)
 
    check_include_file( "pa_jack.h" PA_HAS_JACK )
 
@@ -111,7 +111,7 @@ list( APPEND DEFINES
 
 list( APPEND LIBRARIES
    PRIVATE
-      portaudio::portaudio
+      PortAudio::PortAudio
 )
 
 if( PA_HAS_ALSA )
diff --git a/libraries/lib-audio-devices/CMakeLists.txt b/libraries/lib-audio-devices/CMakeLists.txt
index 997ffc61b..60992db93 100644
--- a/libraries/lib-audio-devices/CMakeLists.txt
+++ b/libraries/lib-audio-devices/CMakeLists.txt
@@ -21,7 +21,7 @@ set( SOURCES
    Meter.h
 )
 set( LIBRARIES
-   portaudio::portaudio
+   PortAudio::PortAudio
    $<$<BOOL:${USE_PORTMIXER}>:portmixer>
    lib-preferences-interface
 )
